### ISSUE TITLE
Deprecation warning:  context_instance argument of render_to_string 

### ISSUE THREAD

**Issue by ad-m** [dominicrodger/django-tinycontent#15]
```
/home/adas/.virtualenvs/poradnia/local/lib/python2.7/site-packages/tinycontent/templatetags/tinycontent_tags.py:29: RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
  context)
````

**Comment by ad-m** [comment]
Example compat bridge: https://github.com/pennersr/django-allauth/blob/master/allauth/compat.py

**Comment by dominicrodger** [comment]
Are you planning to submit a pull request, or are you just noting the issue? If the latter, I'll get to it when I have a moment.

**Comment by ad-m** [comment]
I have issues with debuggging and testing that due pytest (I don't know them enought) eg. unable run ipdb, print anything, so I unable to fix them. 

Yet it is only warning and not fail even on django master.

**Comment by dominicrodger** [comment]
Fixed in 06c5b6f327e16168db9ff2b0eefd52dd833becd0, and released as 0.6.0. Tests now pass on Django's master branch, and we're compatible back to Django 1.5 for supported Python versions.
